algorithms/refinement/parameterisation/configure.py

```diff
@@ -105,6 +105,14 @@
               "of the serialized model small. At the moment, this only has"
               "an effect for crystal unit cell (B matrix) errors."
 
+    trim_scan_to_observations = True
+      .type = bool
+      .expert_level = 1
+      .help = "For scan-varying refinement, scan objects will be trimmed to"
+              "the range of observed reflections if this is true. This avoids"
+              "failures in refinement for cases where the extremes of scans"
+              "contain no data, such as when the crystal moves out of the beam."
```

Comment:
```suggestion
      .help = "For scan-varying refinement, trim scan objects to"
              "the range of observed reflections. This avoids"
              "failures in refinement for cases where the extremes of scans"
              "contain no data, such as when the crystal moves out of the beam."
```
This wording feels like the linguistic equivalent of 
```
if x is True:
  ...
elif x is False:
  ...
```
See also the help text immediately below:
```Set True to write out...```
which might as well just be
```Write out...```